database/backend/caching.json

```diff
@@ -1 +1,52 @@
-[]
\ No newline at end of file
+[
+    {
+        "name": "Aws caching overview",
+        "description": "In computing, a cache is a high-speed data storage layer which stores a subset of data, typically transient in nature, so that future requests for that data are served up faster than is possible by accessing the data’s primary storage location. Caching allows you to efficiently reuse previously retrieved or computed data.",
```

Comment:
```suggestion
        "description": "In computing, a cache is a high-speed data storage layer that stores a subset of data. It's typically transient in nature so that future requests for said data are served up faster than is possible, which then accesses the data’s primary storage location. Caching also allows developers to efficiently reuse previously retrieved or computed data.",
```